src/xenia/gpu/spirv_shader_translator.cc

```diff
@@ -1015,7 +1015,7 @@ spv::Id SpirvShaderTranslator::BitfieldExtract(spv::Id result_type,
   spv::Id base_type = b.getTypeId(base);
 
   // <-- 32 - (offset + count) ------ [bits] -?-
-  if (32 - (offset + count) > 0) {
+  if (32 != offset + count) {
```

Comment:
The previous code is checking this value is within a valid range (0-31). You're effectively checking one edge case - what does this solve?

Reply:
This should be `if (32 > (offset + count))`